src/gtk/window.cpp

```diff
@@ -876,7 +876,7 @@ static long wxTranslateKeySymToWXKey(KeySym keysym, bool isChar)
             break;
 
         case GDK_KEY_KP_Begin:
-            key_code = isChar ? WXK_HOME : WXK_NUMPAD_BEGIN;
+            key_code = WXK_NUMPAD_CENTER;
```

Comment:
We actually don't even need this change, AFAICS (nor the one for wxMSW below), as it already works correctly...

Reply:
Sorry, don't understand you. Why should we put wxk_home in onChar here?

Reply:
What I meant was that we didn't need to change `WXK_NUMPAD_BEGIN` to `WXK_NUMPAD_CENTER` here, as they're one and the same (and same thing with `WXK_CLEAR` in wxMSW).

As for `isChar`, I guess it's never going to be true here as this key code must only be generated for non-char events, but I'm not 100% sure.

Reply:
Just in the starting message of this issue I show that onChar of such events have WXK_HOME key code, and this is 100% confusing.

Reply:
Sorry, I had missed/forgotten it. Anyhow, I have 0 objections to removing the `isChar` test (except that it might be better to do it in a separate commit), I just was commenting on the fact that if we define `WXK_NUMPAD_CENTER` as a synonym for the existing constant, there is no reason to use the synonym instead of the constant itself in the code.

An alternative -- but, in effect, almost indistinguishable -- approach would be to define `WXK_NUMPAD_CENTER` as a new constant and `WXK_NUMPAD_BEGIN = WXK_NUMPAD_CENTER` for compatibility. And then it would make sense update the code.

But this seems a bit like a mix of the two...

Reply:
fixed, please review